core/src/main/java/org/apache/brooklyn/core/mgmt/persist/ClasspathConfigLoader.java

```diff
@@ -18,55 +18,17 @@
  */
 package org.apache.brooklyn.core.mgmt.persist;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.Enumeration;
-import java.util.Map;
-import java.util.Properties;
-
-import org.apache.brooklyn.util.core.ResourceUtils;
-import org.apache.brooklyn.util.exceptions.Exceptions;
-import org.apache.brooklyn.util.stream.Streams;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.Maps;
-
 /**
  * Loads the class-renames from the configuration file on the classpath.
  *
  * @see {@link #DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH}
  */
-public class ClasspathConfigLoader implements ConfigLoader {
-    private static final Logger LOG = LoggerFactory.getLogger(ClasspathConfigLoader.class);
-    private static final String DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH = "classpath://org/apache/brooklyn/core/mgmt/persist/deserializingClassRenames.properties";
+public class ClasspathConfigLoader extends PropertiesConfigLoader {
```

Comment:
I don't think this class pulls its weight anymore. I'd delete it, and just have the thing that constructed it call `new PropertiesConfigLoader(DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH)`.

I might then rename `PropertiesConfigLoader` to indicate that it's loading the properties from the classpath.